packages/server/src/api/controllers/application.ts

```diff
@@ -47,6 +47,9 @@ import {
   PlanType,
   Screen,
   UserCtx,
+  CreateAppRequest,
+  FetchAppDefinitionResponse,
+  type FetchAppPackageResponse,
```

Comment:
What is the need for the `type` keyword here?

Reply:
This is something my editor seems to be when it auto-imports a type. Will look into configuring it not to do it.